data/actions/scripts/other/potions.lua

```diff
@@ -1,158 +1,221 @@
-local ultimateHealthPot = 8473
-local greatHealthPot = 7591
-local greatManaPot = 7590
-local greatSpiritPot = 8472
-local strongHealthPot = 7588
-local strongManaPot = 7589
-local healthPot = 7618
-local manaPot = 7620
-local smallHealthPot = 8704
-local antidotePot = 8474
-local greatEmptyPot = 7635
-local strongEmptyPot = 7634
-local emptyPot = 7636
-
+--[[
+	-- Don't forget to put new potions in actions.xml before adding here!
+	Values of potions are copied from tibia wiki!
+	-- Explanations!
+	removePotions : removing the potion after use. Default : true (optional)
+	addEmptyFlasks : add the empty flask after using! ( only works if removePotions ) Default : true (optional)
+	words : to say after using! Default : "Aaaah..." (optional)
+	healthRate : affects the formula of health/mana added (min * healthRate), (max * healthRate) - Default: 1 (required)
+	manaRate : same as healthRate but, for mana - Default: 1 (required)
+	-- For every potion!
+	[potionId] = {
+		vocations : the vocations that are able to use the potion! (optional)
+		minLevel : minimum level to use the potion (optional)
+		health/mana : (required)
+			min, max : the Formula of adding health (i.e , the potion addes from 150 ~ 200 health) (required)
+		emptyFlask : the empty flask to add to player (optional)
+	}
+]]
+potions = {
```

Comment:
No, it's not local. Variables (without `local` keyword) are by default Global Variables.